react/features/app-settings/functions.native.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import { Dimensions, Platform } from 'react-native';
+
+const IPHONE_OFFSET = 20;
+const IPHONEX_HEIGHT = 812;
+const IPHONEX_OFFSET = 44;
+const IPHONEX_WIDTH = 375;
+
+/**
+* Determines if it's an iPhone X or not to handle notch on the settings
+* screen properly.
+*
+* FIXME: Settings screen uses a custom implementation of safe views
+* to avoid empty areas on the screen when there should be a header rendered.
+* In long term we need to use a generic solution to handle "safe view
+* areas" for various devices. It can be React 0.50's {@code SafeAreaView} or
```

Comment:
I didn't know it already existed in RN! why are we not using it then? wouldn't it be much simpler than what we are doing here now?

Reply:
I posted a note on Stride about how much I don't like how it renders the screen (too much empty space). Let's talk about it later, when we implement something more generic

Reply:
ok